### ISSUE TITLE
 Focus is not on the cookie box when page is opened - accessibility and screenreader issue

### ISSUE THREAD

**Issue by reetagithub** [DSpace/dspace-angular#2491]
**Describe the bug**
When opening the page, the focus should be in the box asking about the cookies. When this is not the case, the box is not detectable at all to one that uses a screenreader. 

You can see for instance https://yle.fi/ to see how this should be done. Especially when examining the page with a screenreader, for instance VoiceOver (Apple) or NVDA (Windows) this is easily understood.

**To Reproduce**
Open the DSpace front page.

**Expected behavior**
The focus should be in the cookie box. The screenreader should start reading the text in the box and wait for the push of a button before continuing to the page.

**Comment by tdonohue** [comment]
Thanks @reetagithub for reporting this!  I agree that's a major accessibility bug. I've labeled this as such and moved it to our 7.6.x maintenance board in search of a volunteer to help fix it.

**Comment by atarix83** [comment]
@tdonohue @hostle83

4science is interested in working on this task and can be done with https://github.com/DSpace/dspace-angular/issues/2718

Estimation is about 4h
